core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -424,9 +430,12 @@ public void deleteFile(EmailMessage f) throws FileSystemException {
 	}
 	@Override
 	public EmailMessage moveFile(EmailMessage f, String destinationFolder, boolean createFolder) throws FileSystemException {
-		ExchangeService exchangeService = getConnection();
+		String folderNameToUse = getFolderNameToUse(destinationFolder);
+		String mailbox = getMailboxToUse(destinationFolder);
+		ExchangeService exchangeService = getConnection(mailbox);
```

Comment:
I think it would be usefull to combine these three elements in a single record-class, so you can retrieve them with a single call.